### PR TITLE
Add draft mp4msv module

### PR DESCRIPTION
**Pull request by sum2012**
Meruru no Atelier Plus - Arland no Renkinjutsushi 3 - Official PlayView  use it

### CODE REVIEW

File: Core/HLE/sceMp4.cpp [hunk 1]
```diff
@@ -455,8 +465,15 @@ const HLEFunction sceAac[] = {
...
+	{0x9CA13D1A, &WrapU_UU<mp4msv_9CA13D1A>,                    "mp4msv_9CA13D1A",               'x', "xx"      },
+
+};
+
 void Register_sceMp4()
 {
 	RegisterModule("sceMp4", ARRAY_SIZE(sceMp4), sceMp4);
 	RegisterModule("sceAac", ARRAY_SIZE(sceAac), sceAac);
-}
+	RegisterModule("mp4msv", ARRAY_SIZE(mp4msv), mp4msv);
```
Comment (unknownbrackets) [inline Core/HLE/sceMp4.cpp:478]: Please double check that old save states still work.

-[Unknown]

File: Core/HLE/sceMp4.cpp [hunk 2]
```diff
@@ -455,8 +465,15 @@ const HLEFunction sceAac[] = {
 	{0XD2DA2BBA, &WrapU_U<sceAacResetPlayPosition>,              "sceAacResetPlayPosition",           'x', "x"      },
 };
 
+const HLEFunction mp4msv[] = {
+	{0x3C2183C7, &WrapU_UU<mp4msv_3C2183C7>,                    "mp4msv_3C2183C7",               'x', "xx"      },
```
Comment (unknownbrackets) [inline Core/HLE/sceMp4.cpp:469]: Should these be removed from the sceMp4 lines, or hooked up there?

-[Unknown]

File: Core/HLE/sceMp4.cpp (hunk 1 above)
Comment (sum2012) [reply Core/HLE/sceMp4.cpp:482]: Fixed

File: Core/HLE/sceMp4.cpp (hunk 2 above)
Comment (sum2012) [reply Core/HLE/sceMp4.cpp:469]: Do nothing so that include in the file is better

**Review by hrydgard — APPROVED** [review]

**Pull request merged**